clang/lib/CodeGen/CGCall.cpp

```diff
@@ -1857,20 +1857,20 @@ void CodeGenModule::getDefaultFunctionFPAccuracyAttributes(
     StringRef Name, llvm::AttrBuilder &FuncAttrs, unsigned ID,
     const llvm::Type *FuncType) {
   for (const auto &M : getLangOpts().FPAccuracyMap) {
-    llvm::StringSet<> FuncOwnAttrs;
     StringRef FPAccuracyVal = llvm::fp::getAccuracyForFPBuiltin(
         ID, FuncType, convertFPAccuracy(M.second));
-    FuncAttrs.addAttribute("fpbuiltin-max-error=", FPAccuracyVal);
+    if (!FPAccuracyVal.empty())
```

Comment:
Should we issue a diagnostic in this case (and the same check below)? An assertion might be better because I think this will only happen if the front end thinks accuracy should be available for this function but it isn't. I'm not sure what the best behavior is here, but it feels like a problem in our error handling somewhere.

Reply:
May be adding an assertion is the safest. Adding it.